examples/openshift-lightspeed-tls.yaml

```diff
@@ -38,7 +38,7 @@ spec:
           secret:
             secretName: lightspeed-certs
       containers:
-      - image: quay.io/openshiftdemos/lightspeed-w-rag:latest
+      - image: quay.io/openshift/lightspeed-service-api-rag
```

Comment:
:latest?

Reply:
good catch, yes (though i believe it also defaults to latest)